public/app/core/components/Page/Page.tsx

```diff
@@ -109,10 +109,22 @@ const getStyles = (theme: GrafanaTheme2) => {
       margin: theme.spacing(0, 0, 0, 0),
 
       [theme.breakpoints.up('md')]: {
-        margin: theme.spacing(2, 2, 0, config.featureToggles.dockedMegaMenu ? 2 : 1),
-        padding: theme.spacing(3),
+        margin: theme.spacing(2, 2, 0, config.featureToggles.dockedMegaMenu ? 2 : 1), // removed by dockedMegaMenu toggle
+        padding: theme.spacing(3), // changed by dockedMegaMenu toggle
       },
     }),
+
+    // Only applied when the dockedMegaMenu feature toggle is enabled
+    newPageInner: css({
```

Comment:
we could just pass the value of `dockedMegaMenu` to `getStyles`, then we wouldn't have to modify the eslint rule? 🤔